bin/e2e-test-runner

```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
 set -e
 
+export PGHOST=localhost
```

Comment:
```suggestion
export PGHOST="${PGHOST:=localhost}"
```
I _think_ that if we define the variables like this we don't override someone's settings if they have them. Then we can use them in the database URLs below too